constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -109,6 +109,16 @@ public ConstructorIO(String apiToken, String apiKey, boolean isHTTPS, String hos
       this(apiToken, apiKey, isHTTPS, host, null);
     }
 
+    /**
+     * Sets apiKey
+     *
+     * @return true
+     */
+    public boolean setApiKey(String apiKey) {
+        this.apiKey = apiKey;
+        return true;
```

Comment:
why return true?

Reply:
What does Java return by default? Just remove this?

Reply:
> why return true?

Should this just be removed?